ledger-core/virtual/integration/sender_test.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package integration
+
+import (
+	"context"
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/gojuno/minimock/v3"
+	"github.com/stretchr/testify/assert"
+
+	"github.com/insolar/assured-ledger/ledger-core/insolar/jet"
+	"github.com/insolar/assured-ledger/ledger-core/insolar/node"
+	"github.com/insolar/assured-ledger/ledger-core/insolar/payload"
+	"github.com/insolar/assured-ledger/ledger-core/instrumentation/inslogger"
+	"github.com/insolar/assured-ledger/ledger-core/pulse"
+	"github.com/insolar/assured-ledger/ledger-core/reference"
+	"github.com/insolar/assured-ledger/ledger-core/virtual/authentication"
+	"github.com/insolar/assured-ledger/ledger-core/virtual/integration/utils"
+)
+
+var messagesWithoutToken = []struct {
+	name string
+	msg  interface{}
+}{
+	{
+		name: "VCallRequest",
+		msg:  &payload.VCallRequest{},
+	},
+	{
+		name: "VCallResult",
+		msg:  &payload.VCallResult{},
+	},
+	{
+		name: "VStateRequest",
+		msg:  &payload.VStateRequest{},
+	},
+	{
+		name: "VDelegatedCallResponse",
+		msg:  &payload.VDelegatedCallResponse{},
+	},
+	{
+		name: "VFindCallRequest",
+		msg:  &payload.VFindCallRequest{},
+	},
+
+	{
+		name: "VStateReport",
+		msg:  &payload.VStateReport{},
+	},
+	{
+		name: "VDelegatedCallRequest",
+		msg:  &payload.VDelegatedCallRequest{},
+	},
+
+	{
+		name: "VFindCallResponse",
+		msg:  &payload.VFindCallResponse{},
+	},
```

Comment:
Can you make the same order as in TestRail, please ^^

Reply:
done